### ISSUE TITLE
Hosted service (MX servers) are not working

### ISSUE THREAD

**Issue by ahmetb** [niftylettuce/forward-email#14]
I'm pointing my MX records with instructions provided at https://forwardemail.net/ (you can query `ahmet.im` to see my MX/TXT records). However all emails are bouncing.

`I'm getting connection timeouts to `mx1.forwardemail.net`'s IP address from https://mxtoolbox.com/SuperTool.aspx?action=smtp%3a192.81.219.215&run=toolpage.

Is this hosted service actively being supported? I want to trust it with all my email completely but I'd like to understand if there's anyone using the hosted service seriously (it's been down for a week for me) so that perhaps I should do self-hosting myself.

**Comment by advaith1** [comment]
I had this problem too, it used to work but it no longer works.

> The recipient server did not accept our requests to connect. Learn more at https://support.google.com/mail/answer/7720 [mx1.forwardemail.net. 192.81.219.215: timed out] [mx2.forwardemail.net. 192.81.219.215: timed out]

**Comment by niftylettuce** [comment]
The service was abused by a spammer and currently waiting on Spamhaus to remove us from the blocklist

Will probably get new servers set up tonight, upstream provider disabled networking and are incredibly slow with response times, very disapointed

**Comment by ahmetb** [comment]
I am actually planning to move ALL my email to this, and hoping to create e-mail addresses in form of `[service]@mydomain.com`, such as `bankofamerica@ahmet.im`.

Would your recommendation be self-hosting? It looks like there's serious outages and risks of abuse (not to mention potential security issues) of the hosted service?

**Comment by ahmetb** [comment]
Hmm, does gSuite have catch-all + forward? This might be what I'm looking for after all.

**Comment by niftylettuce** [comment]
@wei The purpose of this is to eliminate a middleman that is not transparent about their stack and forwarding.  @ahmetb I recommend to use this service (I use it myself for everything).  The only reason why we had an outage was because I had to temporarily disable spam check because some of the users of this service were having issues and we were debugging (see the commit log).  This allowed someone to abuse the system in the short interim (see blacklist that now exists).

**Comment by ahmetb** [comment]
I see, however the risk (and the precedence) of a week-long outage sounds scary to me as I get some 2FA codes, personal communication and password reset emails.

**Comment by niftylettuce** [comment]
Will do my best to make sure this doesn't happen again.  The most
unfortunate thing is that Digital Ocean did not even alert or forewarn me
that they were disabling networking.  Extremely unprofessional of them and
I'm considering switching to Vultr.com now.

**Comment by ahmetb** [comment]
I recommend switching to a professional cloud provider. I personally work at Google Cloud Platform, feel free to try it out and happy to help if you try to migrate.

**Comment by wei** [comment]
@ahmetb GCP charges a lot more though. 

As an example for a 1 CPU 1 GB RAM 25 GB SSD Virtual Machine

## GCP  -  $146.35 / month
![image](https://user-images.githubusercontent.com/5880908/40932949-0286c480-67fe-11e8-8077-17c205dfb3d1.png)

## Vultr and DO  -  $5 / month
![image](https://user-images.githubusercontent.com/5880908/40932862-b19ee0f2-67fd-11e8-8d4d-5c3f16d875c4.png)

**Comment by ahmetb** [comment]
Yeah, the Premium Network Tier especially will be expensive. That's what's shown there. There's ways to downgrade the networking to Standard Tier (which is what providers like DigitalOcean do). Regardless, you probably won't reach the $5~ ballpark you'd get from Virtual Private Hosting providers.

**Comment by niftylettuce** [comment]
I'm going to switch to Vultr.com and set up a few MX servers (instances of this repo) instead of just one.  Right now mx1 and mx2 are the same IP.  Will also get a status page up and document it when I can.

**Comment by niftylettuce** [comment]
If you try again it should be fine.  Let me know otherwise!  If you want to forward me any rejections/issues/bounced mail you can at niftylettuce@gmail.com for me to further investigate.  I fixed quite a few issues recently.

**Comment by ctindel** [comment]
Is it still down?  I don't even see mx1 listening on port 25.

**Comment by niftylettuce** [comment]
It's up and working fine

**Comment by ctindel** [comment]
@niftylettuce Hmmmm OK.  I'll open a separate issue for this problem.  Thanks!
